package.json

```diff
@@ -43,6 +43,20 @@
       "@commitlint/config-conventional"
     ]
   },
+  "lint-staged": {
+    "*.js": [
+      "eslint --fix",
+      "git add"
+    ],
+    "*.ts": [
+      "prettier --write",
+      "git add"
+    ],
+    "package.json": [
+      "node cli.js",
```

Comment:
This is hilarious 😆

Reply:
I tried to use stable `sort-package-json`, but seems npm can't install package with the same name? 

Should we swith to yarn?

Reply:
This is fine, it's like a little integration test for us 😄

Reply:
Okay